values.yaml

```diff
@@ -216,7 +216,7 @@ nboost:
 
 persistence:
   storageClass: ""
-  pvcSize: 1Gi
+  pvcSize: 10Gi
```

Comment:
This is too low for sterling's enforced value , files can get quite large  current usage in a BB instance  i have is 
```
16G     share/data
17G     share
airflow@search-scheduler-#-:/opt/airflow$ du -h share
```

Reply:
increased to 24G